ChatApp/ChatApp.Core/Entities/User.cs

```diff
@@ -1,19 +1,11 @@
-﻿using ChatApp.Core.Enums;
+﻿using ChatApp.Core.DTO;
 using System.Collections.Generic;
 
 namespace ChatApp.Core.Entities
 {
-    public class User
+    public class User : UserDTO
```

Comment:
I am not sure that it is good decision when domain entity inherits from data transfer object.

Reply:
Done!